### PR TITLE
WFLY-9449 SingletonPartitionTestCase can fail intermittently

### PR DESCRIPTION
**Pull request by rhusar**
Jira
https://issues.jboss.org/browse/WFLY-9449

### CODE REVIEW

**Comment by stuartwdouglas** [comment]
@pferraro please approve

**Comment by rhusar** [comment]
Just to mention, reduces time taken by the test by up to 2 minutes (due to https://issues.jboss.org/browse/WFLY-9420 ).

**Review by pferraro — APPROVED** [review]
I don't love the use of a static servlet property - but I suppose it's fine given the context.

File: testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/singleton/SingletonPartitionTestCase.java [hunk 1]
```diff
@@ -75,6 +75,7 @@
 public class SingletonPartitionTestCase extends ClusterAbstractTestCase {
 
     private static final long TOPOLOGY_CHANGE_TIMEOUT = 120000; // maximum time in ms to wait for cluster topology change
+    private static final int SERVICE_TIMEOUT = 5_000; // it takes a little extra time after merge for the singleton service to migrate
```
Comment (kabir) [inline testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/singleton/SingletonPartitionTestCase.java]: TimeoutUtil.adjust()?

File: testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/singleton/partition/PartitionerServlet.java [hunk 2]
```diff
@@ -36,25 +38,36 @@
...
+ * test suite. While FD/VERIFY_SUSPECT/GMS can be be configured to detect partitions quickly the MERGE3 handles merges within randomized
+ * intervals and uses unreliable channel which can easily take several minutes for merge to actually happen. Also, speeds up the test
+ * significantly.
  *
  * @author Radoslav Husar
  */
 @WebServlet(urlPatterns = {PartitionerServlet.SERVLET_PATH})
 public class PartitionerServlet extends HttpServlet {
 
+    private static final long VIEWS_TIMEOUT = 3_000;
```
Comment (kabir) [inline testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/singleton/partition/PartitionerServlet.java]: TimeoutUtil.adjust()?

File: testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/singleton/SingletonPartitionTestCase.java (hunk 1 above)
Comment (rhusar) [reply testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/singleton/SingletonPartitionTestCase.java]: Done, thanks.

File: testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/singleton/partition/PartitionerServlet.java (hunk 2 above)
Comment (rhusar) [reply testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/singleton/partition/PartitionerServlet.java]: This is running on server side and thus it has no knowledge of the passed in timeout property. Also, this is really not necessary.

**Pull request merged**